06_modules_lab/02.py

```diff
@@ -1,4 +1,15 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
+#!/usr/bin/python
+import sys
 
+if (len(sys.argv)!=3):
+    print "wrong usage"
+
+else:
+    num1 = sys.argv [1]
+    num2 = sys.argv [2]
+    if not (str(num1).isdigit()):
+        print "invalid"
+    elif not (str(num2).isdigit()):
+        print "invalid"
```

Comment:
It's better to use `sys.exit` to quit from a python script with an error - `sys.exit('Invalid input')`